src/util/auth.js

```diff
@@ -67,6 +67,63 @@ function useAuthProvider() {
     return user;
   };
 
+  const setUpRecaptcha = () => {
+    window.recaptchaVerifier = new firebase.auth.RecaptchaVerifier(
+      "recaptcha-container",
+      {
+        size: "invisible",
+        callback: function (response) {
+          console.log("Captcha Resolved");
+        },
+        defaultCountry: "IN",
```

Comment:
Should probably change the default country to be Canada since we're launching in Canada first (incase we somehow decide to launch in more countries after that)

Reply:
Its set to "CA"